src/reducers/toRead.js

```diff
@@ -0,0 +1,10 @@
+import { ADD_TOREAD } from '../actions'
+
+export const toReadList = (state = [], action) => {
```

Comment:
I think we will need to refactor to use the spread operator here so that when the user adds new books to their to Read list without overwriting their original book.